src/App.jsx

```diff
@@ -10,6 +10,7 @@ import TaskList from "./components/TaskList/TaskList";
 const App = () => {
   const [selectedList, setSelectedList] = useState("Inbox");
   const taskLists = {
+    noDate: "",
```

Comment:
Bu ne için kullanılıyor?

Reply:
Tarihsiz tasklar için başlık bilgisine gerek olmayacaktır diye boş bırakmıştım ama "No Date" olarak güncelleyebilirim boş string'i.